app/util/github.py

```diff
@@ -1,48 +1,34 @@
 import requests
-import os, glueops.setup_logging, traceback, json
+import os, glueops.setup_logging, traceback
 
 # Configure logging
 LOG_LEVEL = os.getenv("LOG_LEVEL", "INFO")
 logger = glueops.setup_logging.configure(level=LOG_LEVEL)
 
-class ReleaseWatcher:
-    def __init__(self, url="https://api.github.com/repos/glueops/codespaces/releases?per_page=1"):
-        """
-        Initialize the ReleaseWatcher with a default URL to check for new releases.
-        Stores internal state for the last tag and last assets.
-        """
-        self.url = url
-        self.last_tag = None
-        self.last_assets = set()
+def get_codespace_releases(environment):
+    """
+    Checks GitHub API for releases.
+    Fetches the latest 5 stable releases and the latest 5 releases (including prereleases).
+    """
+    try:
+        response = requests.get('https://api.github.com/repos/glueops/codespaces/releases?per_page=10')
```

Comment:
I think this only does releases. prerelease is false in the request....

<img width="473" alt="image" src="https://github.com/user-attachments/assets/32fa14e3-7f38-4cbc-bc31-18ececc33d0b" />